api/helper/responses.helper.js

```diff
@@ -0,0 +1,45 @@
+module.exports.responseHelper = {
+  responseMapper: (rawData) => {
+    try {
+      let isArray = true;
+      if (!Array.isArray(rawData)) {
+        rawData = [rawData];
+        isArray = false;
+      }
+      const fixedData = rawData.map((data) => {
+        const keys = Object.keys(data);
+        const formattedObj = keys.reduce((structuredData, key) => {
+          const words = key.split("_");
+          const camelCasedWords = words.map((word, index) => {
+            if (index > 0) {
+              const chars = word.split("");
+              chars[0] = chars[0].toUpperCase();
+              return chars.reduce((newKey, char) => {
+                newKey += `${char}`;
+                return newKey;
+              }, "");
+            } else {
+              return word;
+            }
+          });
+
+          const camelCasedKey = camelCasedWords.reduce(
+            (camelCasedWord, word) => {
+              camelCasedWord += word;
+
+              return camelCasedWord;
+            },
+            ""
+          );
+
+          structuredData = { ...structuredData, [camelCasedKey]: data[key] };
+          return structuredData;
+        }, {});
+        return formattedObj;
+      });
+      return isArray ? fixedData : fixedData[0];
+    } catch (error) {
+      return rawData;
```

Comment:
Just do nothing when there is an error. Colour me intrigued. If the called could handle the `rawData` then why bother with this function at all?